rstudio/rstudio.sh

```diff
@@ -79,6 +79,10 @@ if [[ "${ROLE}" == 'Master' ]]; then
   # Install RStudio Server
   REPOSITORY_KEY=95C0FAF38DB3CCAD0C080A7BDC78B2DDEABC47B7
   run_with_retries apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys ${REPOSITORY_KEY}
+  # https://cran.r-project.org/bin/linux/ubuntu/
+  if [[ "${OS_ID}" == "ubuntu" ]]; then
+    wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | sudo tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc
```

Comment:
Use `curl -fsSL --retry-connrefused --retry 10 --retry-max-time 30 ... | ...` for consistency with other init actions and better reliability.

Reply:
Updated